### PR TITLE
OSDOCS-11753: Update the Networking DNS documentation with more accur…

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.13+

Issue:
[OSDOCS-11753](https://issues.redhat.com/browse/OSDOCS-11753)

Link to docs preview:
[DNS](https://80756--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.html#virt-example-nmstate-IP-management-dns_k8s_nmstate-updating-node-network-config)

- [x] SME has approved this change.
- [x] QE has approved this change (Gris Ge).

### CODE REVIEW

File: modules/virt-example-nmstate-IP-management.adoc [hunk 1]
```diff
@@ -134,81 +135,79 @@ spec:
...
+      auto-dns: true
 # ...
 ----
 
-* Configure a static DNS for a network interface with a static IP configuration. Note that for this configuration, you must set the `dhcp` parameter to `false` and the `autoconf` parameter to `false`.
+* If you need to configure a network interface to store DNS values instead of adopting the default method, which uses the `nmstate` API to store DNS values globally, you can set static DNS values and static IP addresses in the network interface YAML file.
++
+[IMPORTANT]
+====
+Storing DNS values at the network interface level might cause name resolution issues after you attach the interface to network components, such as the Open vSwitch (OVS) bridge, Linux bridge and
```
Comment (cathay4t) [inline modules/virt-example-nmstate-IP-management.adoc:163]: and bond?

Comment (dfitzmau) [reply modules/virt-example-nmstate-IP-management.adoc]: Thanks, updated.

**Review by cathay4t — APPROVED** [review]
LGTM

File: modules/virt-example-nmstate-IP-management.adoc [hunk 2]
```diff
@@ -134,81 +135,79 @@ spec:
...
-    next-hop-address: 2001:db8:1::3
-    next-hop-interface: eth1
+      dhcp: false
+      enabled: true
+      autoconf: false
 # ...
 ----
 
-* Configure a static DNS name server to append to Dynamic Host Configuration Protocol (DHCP) and IPv6 Stateless Address AutoConfiguration (SLAAC) servers.
+* If you want to set a static DNS search domains and dynamic DNS name servers for your network interface, define the dynamic interface that runs either the Dynamic Host Configuration Protocol (DHCP) or the IPv6 Autoconfiguration (`autoconf`) mechanism in the network interface YAML configuration file.
```
Comment (kcarmichael08) [inline modules/virt-example-nmstate-IP-management.adoc:199]:
```suggestion
* If you want to set static DNS search domains and dynamic DNS name servers for your network interface, define the dynamic interface that runs either the Dynamic Host Configuration Protocol (DHCP) or the IPv6 Autoconfiguration (`autoconf`) mechanism in the network interface YAML configuration file.
```

File: modules/virt-example-nmstate-IP-management.adoc [hunk 3]
```diff
@@ -225,6 +224,7 @@ interfaces:
 # ...
 ----
 
+
```
Comment (kcarmichael08) [inline modules/virt-example-nmstate-IP-management.adoc:227]: remove empty line?

**Review by kcarmichael08 — COMMENTED** [review]
Looks great! A couple of items for your consideration but lgtm!

File: modules/virt-example-nmstate-IP-management.adoc (hunk 2 above)
Comment (dfitzmau) [reply modules/virt-example-nmstate-IP-management.adoc:199]: :heavy_check_mark:

File: modules/virt-example-nmstate-IP-management.adoc (hunk 3 above)
Comment (dfitzmau) [reply modules/virt-example-nmstate-IP-management.adoc]: :heavy_check_mark:

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**